### DISCUSSION TITLE
How to create a new bar

### DISCUSSION THREAD

**Discussion by pedrojuarezrossello** [webern/mx#141] [Q&A] [answered]
Hey, I'm recently built the library and I can't manage to create a new bar - I'm able to write one following the example, but I'm stuck when creating another one. Can someone help out please?

**Comment by webern** [comment, marked as answer]
Did you forget to add your example?

The example in the readme uses this code to add a measure. You could do this multiple times to add multiple measures:

```c++
    part.measures.emplace_back( MeasureData{} );
    auto& measure = part.measures.back();
```

**Reply by pedrojuarezrossello** [reply]
Hey Mathew, thanks for this! I've figured it out :)
Originally I didn't add a new staff to the new measure, so adding
 ` measure.staves.emplace_back( StaffData{} )` 
and then adding notes to this new staff made it work.

**Reply by webern** [reply]
Nice! I'm glad it's working.
